nvflare/app_opt/sklearn/data_loader.py

```diff
@@ -58,11 +55,14 @@ def load_data_for_range(data_path: str, start: int, end: int, require_header: bo
 
     if hasattr(reader, "skiprows"):
         data_size = end - start
-        if hasattr(reader, "header") and not require_header:
-            data = reader(data_path, header=None, skiprows=start, nrows=data_size)
-        else:
+        if hasattr(reader, "header") and require_header:
             data = reader(data_path, skiprows=start, nrows=data_size)
+        else:
+            data = reader(data_path, header=None, skiprows=start, nrows=data_size)
```

Comment:
One of the "reader" that we get from "get_pandas_reader" is "read_json",
That method does not take in argument "header", I think we can't pass the "header" argument to it.

Reply:
@mikez3 thanks for the PR, can you remove the "read_json" in line 21 in this PR?